internal/pkg/ecs/run_task_request_test.go

```diff
@@ -209,18 +209,19 @@ func Test_RunTaskRequestFromService(t *testing.T) {
 					Subnets:        []string{"sbn-1", "sbn-2"},
 					SecurityGroups: []string{"sg-1", "sg-2"},
 				}, nil)
-				m.EXPECT().ClusterARN(testApp, testEnv).Return("kamura-village", nil)
 			},
 			wantedRunTaskRequest: &RunTaskRequest{
 				networkConfiguration: ecs.NetworkConfiguration{
 					AssignPublicIp: "1.2.3.4",
-					Subnets:        []string{"sbn-1", "sbn-2"},
 					SecurityGroups: []string{"sg-1", "sg-2"},
 				},
 
 				executionRole: "execution-role",
 				taskRole:      "task-role",
 
+				appName: testApp,
```

Comment:
It just occurred to me that we don't have any unit tests for the `CLIString` method of `RunTaskRequest`.

Would it be possible to add one of to test the new scenario in the PR:
```go
func TestRunTaskRequest_CLIString(t *testing.T) {
	testCases := map[string]struct {
		in     RunTaskRequest
		wanted string
	}{
		"generates cmd when --app and --env is provided with additional --security-groups": {},
	}
}
```

Reply:
I was wondering about that gap myself, I'd be happy to add a test yeah